components/cluster/pcs/pcs.p5m

```diff
@@ -2068,3 +2292,4 @@ file path=usr/sbin/pcs
 file path=usr/share/bash-completion/completions/pcs
 file path=usr/share/man/man1m/pcs.1m
 file path=usr/share/pcs/examples/logrotate.d/pcsd
+file files/README path=usr/share/pcs/examples/README
```

Comment:
Please, move it to other custom files (to the top of the manifest), so that it wasn't accidentally removed on component update.